corehq/apps/sms/migrations/0036_index_cleanup.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0034_auto_20191007_0756_noop'),
+    ]
+
+    operations = [
+        migrations.RunSQL("DROP INDEX IF EXISTS {}".format(index), migrations.RunSQL.noop)
```

Comment:
Are any of these large enough that they should be done concurrently?

Reply:
no, I've actually already removed most of them from ICDS